cli tool missing in devops yaml- new question (#105360)
Azure/azure-content: articles/defender-for-cloud/devops-faq.md

- [Is it possible to block the developers committing code with exposed secrets](#is-it-possible-to-block-the-developers-committing-code-with-exposed-secrets)
- [I am not able to configure Pull Request Annotations](#i-am-not-able-to-configure-pull-request-annotations)
- [What are the programing languages that are supported by Defender for DevOps?](#what-are-the-programing-languages-that-are-supported-by-defender-for-devops) 

> - [I'm getting an error in Azure DevOps saying there's no CLI tool](#im-getting-an-error-in-azure-devops-saying-theres-no-cli-tool)

Reviewer: ```suggestion
- [I'm getting the There's no CLI tool error in Azure DevOps](#im-getting-the-theres-no-cli-tool-error-in-azure-devops)
```

Author: Just a small typo in the word 'error' above- it says errorr

Reviewer: @LianaT i fixed the mistake :)

- Java Script
- Type Script

> ### I'm getting an error in Azure DevOps saying there's no CLI tool

Reviewer: ```suggestion
### I'm getting the There's no CLI tool error in Azure DevOps

```

Reviewer: @LianaT this needs to be committed so that the link will work

> If when running the pipeline in Azure DevOps, you receive the following error: 

Reviewer: ```suggestion
When running a pipeline in Azure DevOps, you may receive the `"no such file or directory, scandir 'D:\a\_msdo\versions\microsoft.security.devops.cli'"` error.
```

> "no such file or directory, scandir 'D:\a\_msdo\versions\microsoft.security.devops.cli'".

Reviewer: ```suggestion
```

> This error is because you are missing the dependency of dotnet6 in the pipeline's yaml file. DotNet6 is a requirement in order for the Microsoft Security DevOps extension to run, so you should include this as a task in your yaml file. See more [here](https://marketplace.visualstudio.com/items?itemName=ms-securitydevops.microsoft-security-devops-azdevops). 

Reviewer: ```suggestion
This error occurs if you are missing the dependency of `dotnet6` in the pipeline's YAML file. DotNet6 is required to allow the Microsoft Security DevOps extension to run. Include this as a task in your YAML file to eliminate the error.
 
You can learn more about [Microsoft Security DevOps](https://marketplace.visualstudio.com/items?itemName=ms-securitydevops.microsoft-security-devops-azdevops). 
```

Reviewer: @LianaT these corrections are there to fit to Microsoft standards, please commit
